### DISCUSSION TITLE
Pylontech Speicher + Huawei 4850G2 ohne Victron

### DISCUSSION THREAD

**Discussion by Phantomias2006** [hoylabs/OpenDTU-OnBattery#260] [General]
Hallo Liebe Community,

ich besitze einen HM-1500 mit 4x 405W Panele.
Das Ganze wird von der openDTU ausgelesen und mittels MQTT und NodeRed habe ich mir eine kleine Steuerung für meinen Entfeuchter im Keller gebastelt. 
Aktuell habe ich am Tag ca. 2-5kw Überschuss, welchen ich ins Netz einspeise.
Durch Zufall bin ich auf dieses Projekt aufmerksam geworden. Ich möchte mit dem PV-Überschuss mittels 220V den Pylontech Speicher laden. Ist dies hier möglich ohne einen Victron und nur mit dem Huawei 4850G2 oder nicht?

LG
Florian

**Comment by spcqike** [comment]
Soweit ich das verstanden habe, ja ist möglich. 

Allerdings kannst du nicht sowohl die Batterie als auch die Paneele gleichzeitig am Eingang des wechselrichters anschliessen. 

Du brauchst also entweder zwei WR oder eine Umschaltung am Eingang.

**Reply by Phantomias2006** [reply]
Hi, danke für deine Antwort.
Es wäre sowieso ein zweiter bzw. eigener Wechselrichter für das einspeisen geplant. 
Was mach hier stutzig gemacht hat ist dass das laden bzw. entladen des Speichers als Fallback die Spannungsversorgung des PV Speichers nimmt falls CAN nicht angeschlossen oder keine Kommunikation. Dieser Wert kann dann aber nur vom Victron kommen und deshalb bin ich mir nicht sicher wie sich das System ohne verhält…

**Reply by spcqike** [reply]
Der wechselrichter erkennt eine DC Spannung am Eingang. Das geht natürlich nur korrekt bei 24V oder 48V Batterien. Da er mit einer 12V Batterie nicht startet / läuft. Da bräuchte man einen step-up wandler, wodurch natürlich nicht mehr die Batteriespannung am wechselrichter gemessen werden kann.

**Reply by Manos1966** [reply]
Hoymiles HM-1500
as @spcqike said, Anlaufspannung (V) ist 22V 
Danach kann er runter bis 16V und hoch bis 60V gehen. 
![Hoymiles Spannungsbereich](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/07eb3063-664f-403f-a1cd-462f307f0d64)

**Reply by Manos1966** [reply]
> Allerdings kannst du **nicht** sowohl die Batterie als auch die Paneele gleichzeitig am Eingang des wechselrichters anschliessen.

@spcqike Du sparst mir einen Test (und vermuetlich einen Wechselrichter-Schaden), aber warum nicht?
Ich hatte vor, die Eingeange einer Seite (separates MPTT) mit PV und die zwei Eingaenge der anderen Seite mit der Batterie anzuschliessen.

**Reply by spcqike** [reply]
Weil du pro Eingang nur 375W erzeugen kannst. Und du die Eingänge nicht unabhängig voneinander belasten kannst. 

batterie leer? Du musst den wechselrichter deaktivieren um die Batterie nicht zu schädigen (ja das BMS würde die Batterie trennen. Trotzdem blöd. Auch das wieder verbinden wird für den WR auf Dauer nicht gut sein) 


Mit nur einer Seite kommst du auf 750W. Also geht nachts „nur“ 750W, tagsüber ebenfalls. Außer halt aus der batterie. Aber wenn die von der AC Seite geladen werden soll, wo soll da der Überschuss der 4 Paneele herkommen? 

Und die Paneele direkt auf die Batterie und die an die MPPT erklärt sich von selbst, dass das nicht ideal ist :) 

ne Runde saubere Lösung wird es nur mit batterie, Paneelen und wechselrichter so nicht.

**Reply by Manos1966** [reply]
@spcqike 
Der Grundgedanken der dahinter steckt: PV Panele heute kriegt man fuer 120EUR, VICTRON MPPTs leider nicht.
Ein "kleiner" VICTRON MPPT kann 2-3 PV Panels aushalten, mehr nicht. Ein grosserer VICTRON MPPT 150/45 ist teuer, grosser als das fast "unbezahlbar".
Dann wurde ich lieber (**es ist nur ein Gedanken, deswegen wollten wir testen**) mehr PV Panels kaufen, anstatt teure(rer) VICTRONs.
Vor allem im Winter, bei mageren PV Leistung (ich haette zwei PV Panels mehr als nur VICTRON), 

Du hast Recht natuerlich, "Mit nur einer Seite kommst du auf 750W. Also geht nachts „nur“ 750W"

"tagsüber ebenfalls. Außer halt aus der batterie." Na ja, dachte ich bis jetzt es waere die Loesung um den Verbrauch aus der Batterie von 1500W runter auf 750W zu verringern... **ihr seid mir aber zu schnell** :-d ich muss jetzt die Parametern DC Spannung - Start solar passthrough / DC Spannung - Stop solar passthrough verstehen !!!!  (PS. Respekt Jungs! Mein Kopf raucht vom Wochenende testen, nur um festzustellen, dass eine neue Firmware mit (wahrscheinlich) die Loesung, Montags parat steht...)

**Reply by spcqike** [reply]
@Manos1966 so ganz nachvollziehen kann ich deine Argumentation nicht. 

Du kannst ja auch mehrere kleine victrons parallel schalten. 

Dazu: die Angaben am Victron sind ja bspw 100/15. also 100V Eingang (maximal) und 15A Ausgang (ebenfalls maximal). Je nach Modell kann man dann eine 12, 24 oder 48V Batterie laden. (Die Paneel Spannung muss jeweils 5V drüber sein) 

ein smartsolar 150/35 bspw kann 150V am Eingang (also gut 2 Paneele mit 72 Zellen) und 35A am Ausgang. Bei 48V sind das dann 1680W. Im die zu erreichen kann man halt 4 Paneele in 2s2p anordnen. Also 2 strings mit je 2 Paneelen in Reihe. Macht bei meinen Paneelen ~ 80V am Eingang mit insgesamt 20A also ebenfalls 1.600W. 


Die günstigeren 100/30 und 100/50, die wahrscheinlich ebenfalls passen könnten, können keine 48V Systeme laden. (Und passen könnten, weil die maximale VoC im Winter bei -10 grad von Paneel und seinen Koeffizienten abhängt)

**Reply by Manos1966** [reply]
@spcqike Vielen Dank fuer die ausführliche Erklärung! Die  2s2p Anordnung kling ausgezeichnet.

Das wiederum habe ich nicht verstanden:  "Du kannst ja auch mehrere kleine Vctrons parallel schalten."
Ich bin davon ausgegangen, da ich den VE.Direkt Eingang fuer die OpenDTUonBat nutze. Dadurch kann ich nicht mehrere Victrons miteinander anschließen. Habe ich falsch gedacht?

My Panels are 405Wp STC rated output (Pmpp)  
31,2V Rated voltage (Vmpp) at STC
12,96A Rated current (lmpp) at STC
64,4V (2x31,2V), 25,9 A (2x12,96A)
Dann 2s2p wuerde 128,8V and 51,4A bedeuten, oder 128,8V and 25,9A? 
(2x64,4V = 128,8V) und (2x25,9 A=51,4A), it is STC so Victron's 50A max will be sufficient.

**Reply by Manos1966** [reply]
Oh I found it,

> The second number describes the maximum battery charge current that the MPPT can provide to the battery. 

Also nur die erste Nummer (150V in unserem Victron 150/35 Beispiel) ist wichtig (um den Victron nicht kaputt zu machen).
Die zweite (35A) stellt kein Problem da, falls die PV Panels mehr Amperes liefern.

Klingt gut 👍

**Comment by madmartin** [comment]
Meiner Meinung nach ist das beste Preis/Leistungsverhältnis beim Smartsolar 100/20 (der kann 48V Batterien laden). Kostet bei offgridtec 142€ und wäre für Paneele 2s2p mit weniger als 100V Leerlaufspannung  ausreichend. Der Ladestrom wäre auf 20A begrenzt, das sind dann ungefähr 20A*50V=1000W. Die Leistung der Panels wird dann bei praller Sonne nicht unbedingt voll umgesetzt, aber für den Winter oder bedeckte Tage bei wenig Licht wäre das gut.  ( @spcqike du hast da einen Denkfehler drin mit den 80V * 20A = 1600W - die 20A in der Modellbezeichnung sind der max. Ladestrom für die Batterie )

Wem das nicht gefällt könnte einfach weitere SmartSolar 100/20 parallel schalten. 
Ich habe bei einem Händler Solarmodule mit 550W gefunden (Canadian Solar CS6W-550MS) da passen 2 Stück in Reihe genau zu den max. Leistungsdaten eines SmartSolar 100/20 (Leerlaufspannung 49,6V). Wenn die wieder lieferbar sind werde ich mir mindestens 2 davon kaufen.

Experimentieren mit mehr als zwei Panel in Reihe bei den 100V Typen würde ich übrigens nicht machen, denn die Leerlaufspannung geben Solarpanel schon bei sehr geriger Belichtung ab, und ich würde nicht testen wie sich die SmartSolar bei Überspannung verhalten... die Wahrscheinlichkeit daß das beim Versuch der "Garantieabwicklung" als Überspannungsschaden erkannt wird halte ich für hoch.

**Reply by spcqike** [reply]
Ich hoffe ich hatte keinen Denkfehler. In meiner Berechnung liefern zwei Paneele in Reihe ~80V bei ~10A. Also 800W. Zwei strings also 20A bei 1.600W. 

Ich hatte in meinem Beispiel den 150/35 der, nach stepdown auf 48V eben 35A „herausholt“ was dann wieder in ~1.680W mündet. 

Hast du bei deinen 49.6V VoC auch den Winter berücksichtigt? Nicht, dass der laderegler dann hops geht, wenn die Module ~ 8% mehr Spannung liefern (je nach koeffizient halt) 


Ja, man kann auch mehrere 100/20er nehmen. Hat den Vorteil dass im Falle eines Defekts nicht alles aus ist, sondern nur die Hälfte. Und dass man die strings besser mit verschiedenen Ausrichtungen Ausrichtungen betreiben kann. Und zwei davon sind etwas günstiger als ein 150/35 bspw.

**Reply by Manos1966** [reply]
Das wiederum habe ich nicht verstanden: "Du kannst ja auch mehrere kleine Victrons parallel schalten."
Ich bin davon ausgegangen, da ich den VE.Direkt Eingang fuer die OpenDTUonBat nutze. Dadurch kann ich nicht mehrere Victrons miteinander anschließen. Habe ich falsch gedacht?

**Reply by madmartin** [reply]
> Das wiederum habe ich nicht verstanden: "Du kannst ja auch mehrere kleine Victrons parallel schalten." 

Man darf die Victron Smartsolar elektrisch parallel an die Batterie anklemmen, und über Bluetooth ein "VE.Smart-Netzwerk" einrichten. Dann unterhalten die Lader sich per BT untereinander und arbeiten wie ein großes Ladegerät.

> Ich bin davon ausgegangen, da ich den VE.Direkt Eingang fuer die OpenDTUonBat nutze. Dadurch kann ich nicht mehrere Victrons miteinander anschließen. Habe ich falsch gedacht?

Nein, erstmal alles richtig. Aktuell kann man nur ein VE.Direkt seriellen Anschluß verwenden. Mehrere zu unterstützen ist so gut wie in arbeit.... siehe #129 
Das ist aber nur "Kosmetik" für die Statistik, funktionieren wird es mit einem einzigen Anschluß (ich sage das so weil ich meine Erweiterung noch nicht bekommen habe).

**Comment by Phantomias2006** [comment]
Danke für die Zahlreichen Antworten. Ich muss dass mal für mich sortieren. Ich blicke hier noch nicht vollumfänglich durch. @spcqike du sagst die Spannung nimmt er vom Wechselrichter als Fallback wenn CAN von Pylontech nicht verbunden ist? Der Huawei hängt ja an der selben Leitung, also die Spannung die der Wechselrichter messen würde beim Laden ist nicht die Batterie Spannung sondern die Ladespannung des Huawei… Oder verstehe ich da etwas falsch?

**Reply by spcqike** [reply]
Nein, verstehst du richtig. Die Spannung die dann gemessen würde ist die anliegende Ladespannung. Die aber bei ausreichend dicken Kabeln und einer intakten Batterie annähernd der Batteriespannung sein sollte.

**Comment by Phantomias2006** [comment]
Was mir noch aufgefallen ist:
Der Huawei 4850G2 hat einen online und einen offline mode. Im MQTT Topic sehe ich aber nur die Parameter für den online mode. 
Wird die Ladespannung und Leistung um Hintergrund auch initial für den Offline Mode gesetzt? Ansonsten könnte das Ladegerät beim starten ja die falsche Leistung und Spannung ausgeben wenn die CAN Verbindung was hat.
